libs/feature/search/src/lib/filter-dropdown/filter-dropdown.component.html

```diff
@@ -6,5 +6,6 @@
   [selected]="selected$ | async"
   [allowSearch]="true"
   (selectValues)="onSelectedValues($event)"
+  [attr.data-cy-field]="fieldName"
```

Comment:
It's used [here](https://github.com/geonetwork/geonetwork-ui/pull/595/files#diff-62b755b3b91b8ef83ecf4351865a8bbffd1ab931df79274e229d6f565878602aR175) to check more easily which is the field used in each filter. I thought it would make sense to expose some information to Cypress this way, because we can't really read into the `@Input` of each angular component (it's a shame).

I also used it [in the thumbnail component ](https://github.com/geonetwork/geonetwork-ui/pull/595/files#diff-c66bfc75e726adb10f2e9bf859cc5f88d9a2ba63d05ccac151ca58b30320cf28R7)to let Cypress know whether it's showing a placeholder or not.

But maybe it's not a great way to do things... :grimacing:

Reply:
Oh no, that was just a question out of curiosity! :smile:
I think it's a clever way to handle that!